src/olympia/devhub/templates/devhub/includes/macros.html

```diff
@@ -1,11 +1,8 @@
 {% extends "includes/forms.html" %}
 
-{% macro supported_syntax_tip(allowed_markdown, title=None) %}
+{% macro supported_syntax_link() %}
 <p class="syntax-support">
-{# L10n: %s is a list of markdown syntax. #}
-<span class="tooltip" title="{% if title %}{{ title }}{% else %}{{
-      _('Allowed Markdown: %(allowed_markdown)s', allowed_markdown=allowed_markdown)
-    }}{% endif %}">{{ _('Some Markdown supported.') }}</span>
+  <a href="https://extensionworkshop.com/documentation/develop/create-an-appealing-listing/#make-use-of-markdown" target="_blank">{{ _('Some Markdown supported.') }}</a>
```

Comment:
use `settings.EXTENSION_WORKSHOP_URL` rather than hard-coding the domain.

add `rel="noopener noreferrer` to the `<a`